docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/pagination/pagination.stories.js

```diff
@@ -12,6 +12,12 @@ export default {
 export const Pagination = (knobTab) => {
   const generalKnobTab = typeof knobTab === 'string' ? knobTab : 'General';
 
+  const pagerType = radios('Pager type', {
```

Comment:
Opinionated - but we cannot test pager unless we can see the different types. I can remove if required.

Reply:
please remove this as we need this to be 100% as per figma